espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetAttribute.kt

```diff
@@ -0,0 +1,64 @@
+package io.appium.espressoserver.lib.handlers
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import io.appium.espressoserver.lib.handlers.exceptions.NotYetImplementedException
+import io.appium.espressoserver.lib.model.AppiumParams
+import io.appium.espressoserver.lib.model.Element
+import io.appium.espressoserver.lib.model.ViewAttributesEnum
+import io.appium.espressoserver.lib.model.ViewElement
+import io.appium.espressoserver.lib.viewaction.ViewTextGetter
+
+class GetAttribute : RequestHandler<AppiumParams, String> {
+
+    @Throws(AppiumException::class)
+    override fun handle(params: AppiumParams): String? {
+        val attributeName = params.getUriParameterValue("name")
+        if (attributeName == null || attributeName.trim { it <= ' ' }.isEmpty()) {
+            throw AppiumException("Attribute name cannot be null or empty")
+        }
+
+        // Map attributeName to ENUM attribute
+        val dstAttribute: ViewAttributesEnum? = ViewAttributesEnum.values().find {
```

Comment:
we should raise an expiation here if dstAttribute is null, so the type should not be optional

Reply:
It throws an exception afterwards if it doesn't get inside the `dstAttribute?.let { .... }` block

Reply:
Changed it to:

```kotlin
ViewAttributesEnum.values().find {
            attributeName.equals(it.toString(), ignoreCase = true)
        }?.let { ... }

...
// throw exception
```